web/crux-ui/src/components/nodes/edit-node-card.tsx

```diff
@@ -265,8 +265,9 @@ const EditNodeCard = (props: EditNodeCardProps) => {
                 <DyoButton
                   className="px-6 mt-4 ml-4 mr-auto"
                   secondary
+                  danger={node.status === 'outdated'}
                   onClick={onUpdateNode}
-                  disabled={node.status !== 'connected' || node.updating}
+                  disabled={(node.status !== 'connected' && node.status !== 'outdated') || node.updating}
```

Comment:
It would be nice to have a pure function in the models folder which contains this logic.